### PR TITLE
Feat: fast withdrawals

### PR DESCRIPTION
**Pull request by ayush-tanx**

### CODE REVIEW

**Comment by ayush-tanx** [comment]
Took a pull from polygon deposits due to utils dependency.

File: src/brineconnector/client.py [hunk 1]
```diff
@@ -420,3 +423,162 @@ def list_normal_withdrawals(self, params: Optional[ListWithdrawalParams]=None):
         self.get_auth_status()
         r = self.session.get('/sapi/v1/payment/withdrawals/', json=params)
         return r.json()
+    def get_network_config(self):
+        loc = locals()
+        body = params_to_dict(loc)
```
Comment (austinjfi) [inline src/brineconnector/client.py:428]: remove this where body/params isn't required

File: src/brineconnector/client.py [hunk 2]
```diff
@@ -420,3 +423,162 @@ def list_normal_withdrawals(self, params: Optional[ListWithdrawalParams]=None):
...
+            'deposit_blockchain_hash': deposit_blockchain_hash,
+            'deposit_blockchain_nonce': deposit_blockchain_nonce,
+        }
+        r = self.session.post('/sapi/v1/deposits/crosschain/create/', json=body)
+        return r.json()
+
+    def approve_unlimited_allowance_polygon_network(self, coin: str, signer: Account, w3: Web3):
+        network_config = self.get_network_config()
+        polygon_config = network_config['POLYGON']
+        print(polygon_config)
```
Comment (austinjfi) [inline src/brineconnector/client.py:466]: remove this

File: src/brineconnector/utils.py [hunk 3]
```diff
@@ -109,3 +111,22 @@ def format_withdrawal_amount(amount: int, decimals: int, symbol: str):
...
+def filter_cross_chain_coin(config, coin, type):
+    allowed_tokens = config['tokens']
+    allowed_tokens_for_deposit = config['allowed_tokens_for_deposit']
+    allowed_tokens_for_fast_withdrawl = config['allowed_tokens_for_fast_wd']
+
+    if type == 'TOKENS':
+        allowed_token = allowed_tokens[coin]
+    elif type == 'DEPOSIT':
+        allowed_token = next((token for token in allowed_tokens_for_deposit if token == coin), None)
+    elif type == 'WITHDRAWL':
```
Comment (austinjfi) [inline src/brineconnector/utils.py:123]: Typo here. Please test out all cases.

File: src/brineconnector/utils.py [hunk 4]
```diff
@@ -109,3 +111,22 @@ def format_withdrawal_amount(amount: int, decimals: int, symbol: str):
...
+        allowed_token = next((token for token in allowed_tokens_for_fast_withdrawl if token == coin), None)
```
Comment (austinjfi) [inline src/brineconnector/utils.py:124]: typo here.

**Review by austinjfi — CHANGES_REQUESTED** [review]
Please have a look.

File: example/withdrawals.py [hunk 5]
```diff
@@ -36,33 +36,89 @@ def eth_withdrawals():
...
+            login = client.complete_login(ETH_ADDRESS, PRIVATE_KEY)
+            print('Login Successful')
+        # Error: AuthenticationError | requests.exceptions.HTTPError
+        except exception.AuthenticationError as exc:
+            print(exc)
+        except requests.exceptions.HTTPError as exc:
+            print(exc.response.json())
+
+        user_signature = create_user_signature(PRIVATE_KEY, 'testnet')
+        key_pair = get_stark_key_pair_from_signature(user_signature)
```
Comment (austinjfi) [inline example/withdrawals.py:101]: Remove these functions

**Review by austinjfi — APPROVED** [review]
lgtm

**Pull request merged**